WcaOnRails/app/assets/stylesheets/competitions.scss

```diff
@@ -127,6 +127,29 @@ $venue-map-wrapper-height: 400px;
 }
 
 .competitions-table {
+  font-size: 12px;
+
+  tr {
+
+    &.past {
+      a:link,
+      a:visited {
+        color: #369;
+      }
+    }
+  }
+
+  td {
+    white-space: nowrap;
+
+    &.delegates,
```

Comment:
Yeah...removed the padding just above too, because it was being overwritten by bootstrap. Will look into this later.